fl4health/clients/deep_mmd_clients/ditto_deep_mmd_client.py

```diff
@@ -190,8 +287,9 @@ def compute_training_loss(
                 additional losses indexed by name. Additional losses includes each loss component and the global model
                 loss tensor.
         """
-        for layer_loss_module in self.deep_mmd_losses.values():
-            assert layer_loss_module.training
+        if self.beta_global_update_interval == -1:
```

Comment:
Do we only want to run this assertion when it's -1? If it's non-zero, it seems like maybe we'd want to do this assertion at some point too?

Reply:
Only if interval is -1 we want to activate training of Deep MMD kernel and keep it open, otherwise we will set it True update kernel in each interval and set it back again False, so I don't think we need to have this assertion somewhere else for other cases. Though we can have an assertion her for other cases to make sure it not in training mode.